ui/src/app/utils/access-utils.tsx

```diff
@@ -106,21 +106,6 @@ export const getRegistrationTasks = (navigate): RegistrationTask[] => serverConf
       return profile.twoFactorAuthCompletionTime || profile.twoFactorAuthBypassTime;
     },
     onClick: redirectToTwoFactorSetup
-  }, {
```

Comment:
the order of this structure shouldn't matter - please revert

Reply:
I think that would affect the order in the old homepage when DAR FF is false

Reply:
ah ok - that code has been removed :)

Reply:
I swapped the order in my previous PR:
https://github.com/all-of-us/workbench/commit/d746ea4f39854f8b73fa73c6af38e2ad33d15b98#diff-9a246cecc6677240bbb0b36a1598e2cf3871071322f4e5b80b8b247874cfad50 so I revert it back here.
You suggest me to not revert and keep it as it?

Reply:
doesn't matter.  I'm removing this soon too.